modules/google-cloud-storage-configuring-acs.adoc

```diff
@@ -27,7 +27,7 @@ and link:https://cloud.google.com/iam/docs/creating-managing-service-account-key
 . Enter the *Bucket* name in which you want to store the backup.
 . When using a workload identity, check *Use workload identity*. Otherwise, enter the contents of
   your service account key file into the *Service account key (JSON)* field.
-. Select *Test* (*`checkmark`* icon) to confirm that the integration with GCS is working.
-. Select *Create* (*`save`* icon) to create the configuration.
+. Select *Test* to confirm that the integration with GCS is working.
```

Comment:
Does an attribute exist for "GCS"?

Reply:
It is already defined in the module introduction.